### PR TITLE
Prevent occasional failures in selenium tests

### PR DESCRIPTION
**Pull request by jhrcek**
@manstis please check. About once a week selenium test [ProjectLibraryIntegrationTest#importAndBuildProjectFromCustomRepository](http://janhrcek.cz/random-failures/#/class/org.kie.wb.selenium.ui.ProjectLibraryIntegrationTest/method/importAndBuildProjectFromCustomRepository) 
fails due to click on "build" button being blocked by "Project imported successfully" notification.

Work around that by ignoring the "element is not clickable.. because other element obscures it" exception and try clicking again.

### CODE REVIEW

File: business-central-tests/business-central-tests-gui/src/main/java/org/kie/wb/selenium/util/ObstructedClickExceptionHandler.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
+package org.kie.wb.selenium.util;
```
Comment (manstis) [inline business-central-tests/business-central-tests-gui/src/main/java/org/kie/wb/selenium/util/ObstructedClickExceptionHandler.java]: License please.

**Comment by jhrcek** [comment]
@manstis licence added

**Comment by jhrcek** [comment]
Build failure seems unrelated to this PR:
```
11:33:24 [INFO] --- maven-compiler-plugin:3.7.0:compile (default-compile) @ optacloud ---
11:33:24 Unexpected error while executing the Upstream repositories builder for PR builds! Error while executing Maven process, non-zero exit code!
11:33:24 java.lang.RuntimeException: Error while executing Maven process, non-zero exit code!
```

jenkins retest this

**Comment by jhrcek** [comment]
@manstis green, pleas merge

**Pull request merged**